index.js

```diff
@@ -59,3 +77,5 @@ module.exports = function requireAll(options) {
 function identity(val) {
   return val;
 }
+
+function isObject(arg) { return Object.prototype.toString.call(arg).indexOf('Object') !== -1; }
```

Comment:
I did already have a single exported array in my sample directory.  Changing to what you suggested still passed in all release.